packages/fast-viewer-react/helpers/generateSnapshots.ts

```diff
@@ -2,20 +2,20 @@
 import React = require("react");
 import renderer = require("react-test-renderer");
 
-export interface IExampleProps {
+export interface ExampleProps {
     component: React.ReactNode;
     data: any;
     styles: string;
 }
 
-export interface IExample {
+export interface Example {
     title: string;
-    props?: IExampleProps;
+    props?: ExampleProps;
     children?: React.ReactNode;
 }
 
-export interface IExampes {
-    data: IExample[];
+export interface Exampes {
```

Comment:
How do we want to handle the spelling error here: `Exampes` => `Examples`?

Reply:
oh we can correct - this isn't exported.